_rules/focus-returns-trigger-9au0ou.md

```diff
@@ -0,0 +1,188 @@
+		<button type="button" onclick="openDialog('dialog1', this)">Open modal dialog</button>
+		<button type="button">Do nothing</button>
+		<div id="dialog_layer" class="dialogs">
+			<div role="dialog" id="dialog1" aria-labelledby="dialog1_label" aria-modal="true"
+					 class="hidden">
+				<h2 id="dialog1_label" class="dialog_label">Dialog title</h2>
+				<div class="dialog_form">Dialog content.</div>
+				<button type="button" class="close-button topright" aria-label="Close" title="Close" 
+						onclick="closeDialog(this)">
+					&times;
+				</button>
+				<div style="text-align: right;">
+					<button type="button" onclick="closeDialog(this)">Ok</button>
+					<button type="button" onclick="closeDialog(this)">Cancel</button>
+				</div>
+			</div>
+		</div>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+The button that is activated to display the [modal][] gets the [focus][] returned to it when the [modal][] is dismissed using the escape key.
+
+```html
+Code needed!
+```
+
+#### Passed Example 3
+
+The button that is activated to display the [modal][] gets the [focus][] returned to it when the [modal][] is dismissed using the cancel button.
+
+```html
+Code needed!
+```
+
+#### Passed Example 4
+
+The link that is activated to display the [modal][] gets the [focus][] returned to it when the [modal][] is dismissed using the close button in the modal.
+
+```html
+Code needed!
+```
+
+#### Passed Example 5
+
+The link that is activated to display the [modal][] gets the [focus][] returned to it when the [modal][] is dismissed using the escape key.
+
+```html
+Code needed!
+```
+
+#### Passed Example 6
+
+The link that is activated to display the [modal][] gets the [focus][] returned to it when the [modal][] is dismissed using the cancel button.
+
+```html
+Code needed!
+```
+
+#### Passed Example 7
+The page has a list of buttons that is activated to display a [modal][], the [focus][] is returned to the correct button on dismissing the modal.
+
+```html
+Code needed!
+```
+
+### Failed
+
+#### Failed Example 1
+The button that is activated to display the [modal][] gets the [focus][] returned to the item after it when the [modal][] is dismissed.
+
+```html
+Code needed!
+```
+
+#### Failed Example 2
+The button that is activated to display the [modal][] gets the [focus][] returned to the start of the page when the modal is dismissed.
+
+```html
+Code needed!
+```
+
+#### Failed Example 3
+The link that is activated to display the [modal][] gets the [focus][] returned to the item after it when the [modal][] is dismissed.
+
+```html
+Code needed!
+```
+
+#### Failed Example 4
+The link that is activated to display the [modal][] gets the [focus][] returned to the start of the page when the modal is dismissed.
+
+```html
+Code needed!
+```
+
+#### Failed Example 5
+The page has a list of buttons that is activated to display a [modal][], the [focus][] is returned to the incorrect button on dismissing the modal.
+
+```html
+Code needed!
+```
+
```

Comment:
I think we need here an example with several ways to close the modal, and one of them returns the focus but not the other (this illustrate that **all** ways to close the modal are tested).